cds_dojson/marc21/fields/books/multipart.py

```diff
@@ -40,31 +92,38 @@ def title(self, key, value):
             }
 
 
-@model.over('volumes', '^246__')
-def volumes(self, key, value):
+@model.over('_migration', '^246__')
+def migration(self, key, value):
     """Translates volumes titles."""
```

Comment:
I would maybe just add an extra comment to explain why this is needed: not records in CDS, records in ILS.